dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -325,7 +327,7 @@ namespace dwa_local_planner {
     goal_front_costs_.setTargetPoses(front_global_plan);
 
     constexpr double MIN_GOAL_DIST_SQ = 0.7;
-    if (sq_dist > MIN_GOAL_DIST_SQ && path_align_costs_.isTurningRequired()) {
+    if (sq_dist > MIN_GOAL_DIST_SQ) {
```

Comment:
the `&& path_align_costs_.isTurningRequired()` should be kept, because it affects the other critics costs

Reply:
good point, fixed